samples/templates/express-direct-auth/web-server/routes/logout.js

```diff
@@ -8,7 +8,7 @@ router.post('/logout', async (req, res) => {
     const authClient = getAuthClient(req);
     // Get okta signout redirect url
     // Call this method before revoke tokens as revocation clears tokens in storage
-    const signoutRedirectUrl = authClient.getSignOutRedirectUrl();
+    const signoutRedirectUrl = authClient.getSignOutRedirectUrl({});
```

Comment:
This method had been changed to set `{}` as the default param. You can rebase to see the change.